### DISCUSSION TITLE
Best configuration for Genmon - Briggs / Synology / Existing infrastructure and hardware

### DISCUSSION THREAD

**Discussion by southern-junior** [jgyates/genmon#1128] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have submitted logs

### What is/are your question(s)?

I am about to dive into the Genmon world. I am looking at leveraging as much of my existing infrastructure/hardware as possible and would love the groups advice.

What I have available at Briggs install:
-Spare cat 6 cable wired to my network closet (running all ubiquity kit)
-Rigid conduit from Briggs to network closet (currently has single coaxial cable for Xfinity service to modem)

Network closet:
-Synology DS1621+
-Ubiquity UMDSE
-Switch

option 1:
Run 16AWG wire from briggs through rigid conduit to network closet. Attach to USR-W630. Use lan port to connect to RasPi. install genmon as advertised.

option 2: run 16AWG wire from briggs through rigid conduit to network closet. Attach USR-W630 to synology. **given synology is linux based. Run Genmon on Synology**.

option 3: Install USR-W630 to Briggs at site. Use existing Cat6 to connect lan to switch. Then use either pi or synology for genmon.

Looking forward to help. Appreciate it.

**Comment by jgyates** [comment]
Although not officially supported, several users have had success running on docker with a serial over TCP connection. Synology renamed their docker server Container Manager. This should be possible.

Here is a link that shows past issues with folks who have questions / answers about genmon in a docker or VM

https://github.com/jgyates/genmon/issues?q=label%3A%22docker+or+vm%22+

**Reply by southern-junior** [reply]
would you recommend running my comms wire through the conduit to the closet (roughly 60 feet) or using the USR-W630 with the existing Cat6?

worried humid conditions will destroy the pi outdoors.

**Reply by jgyates** [reply]
Since the briggs unit uses RS-485, 60 feet is no problem. I would go with option 1 or 2.  You should be able to connect the LAN port on the USR-W630 to your network switch and access it from the pi or synology over Ethernet since genmon can use serial over IP. Wired connections are less prone for errors due to bad signal quality with wireless so I prefer wired when possible.

Option 1 (using a pi) is more straight forward, option 2 (synolgy hosted docker) requires you to setup the docker (container) on synlogy by using the info posted in the threads above. It really depends on your comfort level with setting up a pi vs a container. The pi setup is well documented.

**Reply by skipfire** [reply]
For the humidity I wouldn't worry, I'm in southeastern Louisiana where 80% is low humidity and I have Pis that are fine for years.  SD cards wear out, but that happens indoors or outdoors.

**Reply by southern-junior** [reply]
Ill need a USR-W630 regardless. I guess the more straight forward approach is the existing Cat6. less new wire to run.

If it will run in LA, SETX is fine then. Thank you for your help and response!
